packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.controller.js

```diff
@@ -0,0 +1,81 @@
+import get from 'lodash/get';
+import { ISSUE_TYPE_ID, TICKET_BASE_URL } from './upgrade.constants';
+
+export default class DedicatedUpgradeController {
+  /* @ngInject */
+  constructor($translate, OvhApiSupport, $stateParams, atInternet, Alerter) {
+    this.$translate = $translate;
+    this.OvhApiSupport = OvhApiSupport;
+    this.$stateParams = $stateParams;
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.issueTypeFieldsStr = this.$translate.instant(
+      'dedicated_server_dashboard_upgrade_label',
+    );
+  }
+
+  cancel() {
+    this.atInternet.trackClick({
+      name: `dedicated::dedicated::server::dashboard::upgrade::${this.$stateParams.selectedUpgrade}::cancel`,
+      type: 'action',
+    });
+    this.goBack();
+  }
+
+  upgrade() {
+    if (this.issueTypeDescription.trim()) {
```

Comment:
Can't we just require the field ?

Reply:
the field is required but the function is called when empty :/